app/views/users/show.html.erb

```diff
@@ -1 +1,45 @@
-<h1>The profile of individual user</h1>
\ No newline at end of file
+<div class="container">
+  <div class="card mb-3" style="max-width: 800px;">
+    <div class="row g-0">
+        <div class="col-md-4">
+            <img src =<%= @user.photo %> class="img-fluid rounded" alt="Profile picture" style="height: 200px; width: 200px;">
+        </div>
+        <div class="col-md-4">
+            <div class="card-body">
+                <h5 class="card-title"><%= link_to @user.name, user_path(@user), style: "text-decoration: none;" %></h5>
+            </div>
+        </div>
+        <div class="col-md-4 d-flex align-items-center">
+            <div class="card-body">
+                <p class="card-text"><small class="text-muted">Number of posts: <%= @user.post_counter %></small></p>
+            </div>
+        </div>
+    </div>
+  </div>
+  <div class="card mb-3" style="max-width: 800px;">
+    <div class="card-body text-success text-bg-light p-3">
+      <h5 class="card-title">Bio</h5>
+      <p class="card-text"><%= @user.bio%></p>
+    </div>
+  </div>
+  <% @recent_posts.each do |post|%>
+    <div class="card mb-3" style="max-width: 800px;">
+      <div class="card-body text-success">
+        <a href = <%= user_post_path(post.author.id, post.id) %> style="text-decoration: none;">
+          <h5 class="card-title">Post #<%= @recent_posts.index(post) + 1%></h5>
+        </a>
+        <p class="card-text"><%= post.text %></p>
+        <div class= "card-text">
+          <span class="card-text">Comments: <%= post.comment_counter%></span>
+          <span class="card-text">Likes: <%= post.like_counter%></span>
+        </div>
```

Comment:
- [ ] I was not able to add comments to the posts, which resulted in an error. Kindly fix this to compliment your brilliant work so far 👍 
###Error view:
![image](https://github.com/zuhebahmed88091/Blog_App_Rails/assets/50990925/f6c29fe9-1a68-47f6-9bd6-249ee21083f5)

Reply:
Very well noted. Good job 👏